PWGGA/GammaConv/macros/AddTask_OmegaToPiZeroGamma_pp.C

```diff
@@ -70,7 +71,7 @@ void AddTask_OmegaToPiZeroGamma_pp(  Int_t     trainConfig                   = 1
                                 TString   cutnumberAODBranch            = "000000006008400001001500000",
                                 Int_t     enableExtMatchAndQA           = 0,                      // disabled (0), extMatch (1), extQA_noCellQA (2), extMatch+extQA_noCellQA (3), extQA+cellQA (4), extMatch+extQA+cellQA (5)
                                 Bool_t    enableV0findingEffi           = kFALSE,                 // enables V0finding efficiency histograms
-                                Int_t     enableTriggerMimicking        = 0,                      // enable trigger mimicking
+                                Bool_t    enableTriggerMimicking        = kFALSE,                 // enable trigger mimicking
```

Comment:
This was changed to an integer for a reason. The trigger mimicking can now be based on our framework (case 1) or on the TriggerMaker (case 2).